packages/components/Column/index.ts

```diff
@@ -0,0 +1 @@
+export { Column } from "./Column";
```

Comment:
I think personally I'd lean towards putting the content of `Column.tsx` here until there is more than one thing in this directory. What are your thoughts?.

Reply:
I'm down with that to keep it simple! 👍 It would be nice to have a larger discussion on how we want to organize here. I know @markerikson mentioned earlier in Discord about how we might import/export. Not sure how many IDEs support [file nesting](https://twitter.com/buildsghost/status/1522281718643789826), but could be nice to standardize on how we store examples, docs, etc.

Reply:
fwiw, I personally _really_ dislike having lots of `index.ts[x]` files in a codebase, and am also not a fan of `components/Component/index.tsx` - I'd rather have `Component.tsx` so that the import name is explicit, and I don't have dozens of `index.tsx` files open in tabs :)

Not gonna make a fuss about it either way, just listing my own preference.

Reply:
Old time `index.js` guy who's now indifferent.

Reply:
Sweet, yeah I like simplifying things! I'll add this to the design system RFC so we can discuss it further :)